Session/Calls/CallVC.swift

```diff
@@ -33,6 +33,8 @@ final class CallVC: UIViewController, VideoPreviewDelegate, AVRoutePickerViewDel
     
     var floatingViewVideoSource: FloatingViewVideoSource = .local
     
+    private var willEndToEnableCameraPermission: Bool = false
```

Comment:
Instead of making this an instance variable, you can just pass it into `endCall` and from there into `shouldHandleCallDismiss`